bitcoin/rpc.py

```diff
@@ -508,41 +472,83 @@ def getblockhash(self, height):
             raise IndexError('%s.getblockhash(): %s (%d)' %
                     (self.__class__.__name__, ex.error['message'], ex.error['code']))
 
-    def getinfo(self):
-        """Return a JSON object containing various state info"""
-        r = self._call('getinfo')
-        if 'balance' in r:
-            r['balance'] = int(r['balance'] * COIN)
-        if 'paytxfee' in r:
-            r['paytxfee'] = int(r['paytxfee'] * COIN)
+    def getblockstats(self, hash_or_height, *args):
+        # On clients before PR #17831, passing hash as bytes will result in Block not found
+        """Return a JSON object containing block stats"""
+
+        try:
+            hval = b2lx(hash_or_height)
```

Comment:
In other cases, the type of the value is checked. I think it would be better to do this the same way here, for consistency.

Reply:
Thanks! I will make this change.